package.json

```diff
@@ -164,6 +163,9 @@
     "webpack-inject-plugin": "^1.5.3",
     "yargs": "^14.0.0"
   },
+  "optionalDependencies": {
+    "@swc/core-linux-x64-gnu": "^1.2.160"
+  },
```

Comment:
CircleCI complained about `Error: Cannot find module '@swc/core-linux-x64-gnu'`.
This is a solution from https://github.com/swc-project/swc/discussions/4152.

Reply:
Do we know what this package actually does or why it's needed?

Reply:
It's binary for `@swc/core`, which is required by `nx`.
It is only missing in CI env, not in local.